server/src/http/feature_refresher.rs

```diff
@@ -515,6 +526,7 @@ mod tests {
             engines_cache,
             duration,
             None,
+            false,
```

Comment:
Won't complain if you don't feel like doing this but this struct has a `Default`, this big chunk of noise where you have to update this in tests a lot of times wouldn't happened if these tests used default + destructuring. If you want to be a good citizen that would prevent this problem coming back (but probably not make this PR any simpler)

Reply:
Hmm do you mean this? https://github.com/Unleash/unleash-edge/pull/474/commits/3407d2d444d685c04fcd66eacce610e1e3a750da

Reply:
Yaaaas